TestApp/ios/Podfile.lock

```diff
@@ -311,9 +311,10 @@ DEPENDENCIES:
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  trunk:
-    - AppCenter
+  https://github.com/CocoaPods/Specs.git:
```

Comment:
Why not keeping trunk for this?

Reply:
Let's keep it as it is for now, as only @russelarms knows the context. We will revert it during the next release.